dev/Qmass_boundaries.py

```
@@ -0,0 +1,78 @@
+        print(
+            f"  Viscosity from PQmass (9e5, 0.0): {AS.viscosity():.12f} Pa.s ({AS.p():.12f} Pa, {AS.T():.12f} K)"
+        )
+
+        AS.update(CP.PQmass_INPUTS, 9e5, 1.0)
+        print(
+            f"  Viscosity from PQmass (9e5, 1.0): {AS.viscosity():.12f} Pa.s ({AS.p():.12f} Pa, {AS.T():.12f} K)"
+        )
+
+        return True
+
+    except Exception as e:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)